practitioner-guides/website-landing.md

```diff
@@ -9,4 +9,6 @@ These guides will be especially useful for Open Source Program Offices (OSPOs),
 * [Practitioner Guide: Responsiveness](https://chaoss.community/practitioner-guide-responsiveness/)
 * [Practitioner Guide: Organizational Participation](https://chaoss.community/practitioner-guide-organizational-participation/)
 
+A short video about each guide can be found in the [Practitioner Guides Playlist](https://www.youtube.com/watch?v=xTK2xsd4jZE&list=PL60k37cxI-HSHV4-rEsWMzExw2y2Oq79Z) in the CHAOSS YouTube channel.
```

Comment:
I think this link should be this: https://www.youtube.com/playlist?list=PL60k37cxI-HSHV4-rEsWMzExw2y2Oq79Z